contrib/win32/openssh/install-sshd.ps1

```diff
@@ -264,10 +264,10 @@ if (Get-Service ssh-agent -ErrorAction SilentlyContinue)
    sc.exe delete ssh-agent 1>$null
 }
 
-New-Service -Name ssh-agent -BinaryPathName $sshagentpath -Description "SSH Agent" -StartupType Manual | Out-Null
+New-Service -Name ssh-agent -BinaryPathName `"$sshagentpath`" -Description "SSH Agent" -StartupType Manual | Out-Null
```

Comment:
no need to quoted by ` here.

Reply:
This is intentional. We want the double quotes to persist in binary path. if you skip `, powershell will eat away the double quotes.